### DISCUSSION TITLE
Updating this to .net 8 ?  Show use of aspire ?

### DISCUSSION THREAD

**Discussion by punkouter26** [spark-dotnet/framework#50] [General]
just wondering.

**Comment by faizalmansor** [comment]
![Screenshot 2025-06-23 100730](https://github.com/user-attachments/assets/56e0c611-54cb-40e8-98fe-f927d4f62d9d)
According to the online doc here https://spark-framework.net/docs/introduction/ it's already updated to .NET 8
